nimble/host/mesh/src/adv.c

```diff
@@ -105,9 +105,14 @@ static inline void adv_send(struct os_mbuf *buf)
 
 	adv_int = max(adv_int_min,
 		      BT_MESH_TRANSMIT_INT(BT_MESH_ADV(buf)->xmit));
+#ifdef MYNEWT_VAL(BLE_CONTROLLER)
```

Comment:
I believe it should be #if MYNEWT_VAL(BLE_CONTROLLER) as we want to check for value 1

Reply:
yes, my bad. will fix